### PR TITLE
remove environment variables in `postinstall` script for Windows (fixes #4)

### LINKED ISSUE #4: postinstall scripts don't run on windows
...

> webvr-demos@1.0.0 postinstall c:\Users\Casey\Developer\webvr-demos
> POLAR_SEA_VIDEOS_URL=http://mozvr.com/projects/polarsea/content/videos/ npm ru
> n syncmedia

'POLAR_SEA_VIDEOS_URL' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "c:\Program Files (x86)\nodejs\node.exe" "c:\Program Files (x8
6)\nodejs\node_modules\npm\bin\npm-cli.js" "install"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code ELIFECYCLE
npm ERR! webvr-demos@1.0.0 postinstall: `POLAR_SEA_VIDEOS_URL=http://mozvr.com/p
rojects/polarsea/content/videos/ npm run syncmedia`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webvr-demos@1.0.0 postinstall script 'POLAR_SEA_VIDEOS_UR
L=http://mozvr.com/projects/polarsea/content/videos/ npm run syncmedia'.
npm ERR! This is most likely a problem with the webvr-demos package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     POLAR_SEA_VIDEOS_URL=http://mozvr.com/projects/polarsea/content/vid
eos/ npm run syncmedia
npm ERR! You can get their info via:
npm ERR!     npm owner ls webvr-demos
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\Users\Casey\Developer\webvr-demos\npm-debug.log

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
...
+    "postinstall": "npm run syncmedia && npm run syncmozvr",
+    "syncmedia": "if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.mp4' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.mp4 'http://mozvr.com/projects/polarsea/content/videos/PolarSeaTrailer-3.mp4' ); fi; if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.webm' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.webm 'http://mozvr.com/projects/polarsea/content/videos/PolarSeaTrailer-3.webm' ); fi",
     "syncmozvr": "mkdir -p public/demos/mozvr-site/ && cd node_modules/mozvr-site/ && sed -e 's/\"url\":.*$/\"url\": \"\\/demos\\/mozvr-site\",/g' config.json > config_local.json && wintersmith build -o ../../public/demos/mozvr-site/ --config config_local.json && rm -f config_local.json && cd -",
-    "release": "release-it"
+    "release": "rm -rfv public/demos/polarsea/videos/* && release-it"
```
Comment (cvan) [inline package.json]: should probably not remove the videos every time we release. should probably use a `.npmignore` instead.

Comment (cvan) [inline package.json]: added.

**Comment by caseyyee** [comment]
There is more than just the environment variables not working unfortunately (damn windows!).   Is there a solution that we can specify a list of files/paths more easily?   Going through the script to add more files is going to be a problem.

**Comment by caseyyee** [comment]
Replaced with gulp tasks.

**Pull request closed without merging**